### PR TITLE
Subedits

### PR DESCRIPTION
**Pull request by rdarshan927**
just changed personal links to public
updated target in the right notice part
linked stylesheets ... no big deal in visibility except the contact page
just to notify

### CODE REVIEW

**Comment by rdarshan927** [comment]
and we'll think about the extra folder later ... it'll be there until then

File: main.css
Comment (nmdra) [inline main.css:1]: Any Reason to adding h3 ?. I think it's unwanted for now. 
If you wanna add it, remove Line 34 (flex Order).

**Review by nmdra — DISMISSED** [review]

**Review by nmdra — CHANGES_REQUESTED** [review]
File: main.css

Any Reason to adding h3 ?. I think it's unwanted for now.
If you wanna add it, remove Line 34 (flex Order).

**Comment by rdarshan927** [comment]
it was just the sketch to include, it's not the actual interface design to be included in the page!
(is it good to add the contribute button in contact or contributers page?)

**Pull request closed without merging**